script/lib/ConfigLib.sol

```diff
@@ -120,3 +122,109 @@ library ConfigLib {
         params.lzChainId = uint32(config.readUint(string.concat(prefix, ".lzChainId")));
     }
 }
+
+/**
+ * @title ConfigUtils
+ * @dev Library for updating configuration files with deployed addresses using proper JSON serialization
+ */
+library ConfigUtils {
+    using stdJson for string;
+
+    Vm constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    /**
+     * @dev Updates the config.json file with a new address for a specific chain
+     * @param chain The chain identifier (e.g., "sepolia", "arbitrum_sepolia")
+     * @param fieldName The field name to update (e.g., "iexecLayerZeroBridgeAddress")
+     * @param value The address value to set
+     */
+    function updateConfigAddress(string memory chain, string memory fieldName, address value) internal {
+        updateConfigAddress(chain, fieldName, value, "config/config.json");
+    }
+
+    /**
+     * @dev Updates the config file with a new address for a specific chain
+     * @param chain The chain identifier (e.g., "sepolia", "arbitrum_sepolia")
+     * @param fieldName The field name to update (e.g., "iexecLayerZeroBridgeAddress")
+     * @param value The address value to set
+     * @param configPath The path to the config file
+     */
+    function updateConfigAddress(string memory chain, string memory fieldName, address value, string memory configPath)
+        internal
+    {
+        // Check if file exists
+        if (!vm.exists(configPath)) {
+            console.log("Config file not found at:", configPath);
+            revert("Config file not found");
+        }
+
+        // Read and validate JSON
+        string memory content = vm.readFile(configPath);
+        _validateJsonContent(content);
+
+        // Convert address to string and create JSON value
+        string memory addressString = vm.toString(value);
+        string memory jsonValue = string.concat('"', addressString, '"');
```

Comment:
Have you considered
```solidity
function serializeAddress(string calldata objectKey, string calldata valueKey, address value)
    external
    returns (string memory json);
```
https://getfoundry.sh/reference/cheatcodes/serialize-json#serializejson